core/deployment/src/main/java/io/quarkus/deployment/dev/testing/LogCapturingOutputFilter.java

```diff
@@ -56,9 +56,7 @@ public boolean test(String logRecord, Boolean errorStream) {
                 synchronized (logOutput) {
                     if (convertToHtml) {
                         ByteArrayOutputStream out = new ByteArrayOutputStream();
-                        HtmlAnsiOutputStream outputStream = new HtmlAnsiOutputStream(out) {
```

Comment:
See here, we weren't closing the stream anyway so the attributes weren't closed.